README.md

```diff
@@ -181,6 +182,7 @@ for all comparison operators. When used on the `=` operator it falls
 back to the patch level comparison (see tilde below). For example,
 
 * `1.2.x` is equivalent to `>= 1.2.0, < 1.3.0`
+* `==1.2.x` is equivalent to `== 1.2.0`
```

Comment:
How is this different from `1.2.0` or `=1.2.0` as the constraint? The `.x` is specifically opting into a range and saying any value for the patch is acceptable. Why not put the exact version you want in instead of an `x`?

If you look at this function you'll see that is used `Equal` if the function isn't dirty (dirty is the internal term for a range character)...

https://github.com/Masterminds/semver/blob/f4a4d83067a06c7875d3547f214ed9a01636b1e8/constraints.go#L490-L508

Reply:
I understand your point, the dirty should not be interpreted in a strict equal

Reply:
Does the new version seems ok to you?